Cargo.toml

```diff
@@ -62,11 +61,12 @@ serde = "1.0"
 serde_json = "1.0"
 syn = { version = "1.0", features = ["full"] } # used in macros, so full doesn't affect binary size
 test-helpers = { path = "./crates/test-helpers" }
+test-helpers-macros = { path = "./crates/test-helpers-macros" }
```

Comment:
Why do we have 2 crates, `test-helpers` and `test-helper-macros` while both crates only contain macros? :)

Reply:
proc-macros need their own crate and have `proc-macro = true` in cargo.toml.